pypi_push.sh

```diff
@@ -0,0 +1,25 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# first bump up package.json manually, commit and tag
+rm airflow/www_rbac/static/dist/*
+cd airflow/www_rbac/static/
+yarn
+npm run build
```

Comment:
yarn _and_ npm?!

Reply:
`yarn` is a replacement of `npm install` here because it is faster and more deterministic when installing packages. We can also use `npm install` but it would be slow.

Reply:
@r4vi that is absolutely possible. The primary manager I'm using is `npm` and only using `yarn` to install the packages because yarn is pretty fast. One can use any of the yarn or npm for this.

Reply:
If we can get away with a single thing that would be cleaner.

Reply:
Okay, I'll update this to use only `npm` to eliminate the confusion.

Reply:
You can go for only Yarn if you'd rather. I don't mind which we use.